katas/pokerhands/charles/play.js

```diff
@@ -0,0 +1,222 @@
+import { PokerHand } from './pokerHand';
+
+export class Play {
+    static rankingPH = {
+        'HC': 1,
+        'PA': 2,
+        '2P': 3,
+        '3K': 4,
+        'ST': 5,
+        'FL': 6,
+        'FH': 7,
+        'FK': 8,
+        'SF': 9,
+    };
+
+    #players;
+
+    constructor(pokerHandsPlayers) {
+        this.#players = {};
+        let playersPH = pokerHandsPlayers.split('  ');
+        let valPlayer;
+        let numPlayer;
+        let cardsPlayer;
+        let countPlayers = 1;
+        playersPH.forEach(player => {
+            valPlayer = player.split(': ');
+            numPlayer = 'player' + countPlayers;
+            cardsPlayer = new PokerHand(valPlayer[1].split(' '));
+            this.#players[numPlayer] = {
+                'name': valPlayer[0],
+                'pokerHand': cardsPlayer,
+                // 'rankPH': this.rankingPokerHand(valPlayer[1].split(' '))
+                'rankPH': this.rankingPokerHand(cardsPlayer)
+            };
+            countPlayers++;
+        });
+    }
+
+    pokerHandsPlayers() {
+        return this.getPlayers();
+    }
+
+    classificationResult() {
```

Comment:
This method is a little bit hard to read and understand, does it cover all possible cases for card suits and value combinations?